origami/models/rtu/channels/files.py

```diff
@@ -60,7 +60,7 @@ class FileSubscribeRequest(FilesRequest):
 # - List of deltas to squash into the NotebookBuilder immediately
 class FileSubscribeReplyData(BaseModel):
     deltas_to_apply: List[FileDelta]
-    latest_delta_id: uuid.UUID
+    latest_delta_id: Optional[uuid.UUID]
```

Comment:
Why is this optional now?

Reply:
I think I observed this in either e2e tests or click testing